src/store/global/global.thunks.js

```diff
@@ -22,7 +22,7 @@ function setHermezEnvironment () {
     hermezjs.TxPool.initializeTransactionPool()
 
     if (!window.ethereum) {
-      return dispatch(globalActions.loadEthereumNetworkSuccess({ name: 'mainnet' }))
+      return dispatch(globalActions.loadEthereumNetworkSuccess({ name: '' }))
```

Comment:
Dispatching `''` as the name wouldn't render the button with nothing inside? Or am I wrong? Maybe dispatching `undefined` instead and then render the button only if `name !== undefined` would be better?